.github/workflows/ghcr.yml

```diff
@@ -1,26 +1,37 @@
-name: GHCR
+name: Build Base Enketo in GCHR and Trigger Extra Widgets 
 
 on:
   workflow_dispatch:
   push:
     branches: [main]
-    tags:
+    tags: ['*']
 
 env:
     REGISTRY: ghcr.io
     IMAGE_NAME: ${{ github.repository }}
 
 jobs:
-    build:
+    build-base:
         runs-on: ubuntu-latest
         permissions:
             contents: read
             packages: write
-
+        outputs:
+          image-tag: ${{ steps.first-tag.outputs.tag }}
+          short-sha: ${{ steps.sha.outputs.short }}
+          branch: ${{ steps.branch.outputs.name }}
         steps:
             - name: Check out repository
               uses: actions/checkout@v4
-
+              
+            - name: Get short SHA
+              id: sha
+              run: echo "short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+      
+            - name: Get branch name
+              id: branch
+              run: echo "name=${GITHUB_REF_NAME}" >> $GITHUB_OUTPUT
+      
             - name: Store version information
               run: git describe --tags > .tag.txt || git rev-parse --short HEAD > .tag.txt
```

Comment:
FYI latest run has
> fatal: No names found, cannot describe anything.

questions:
1. is that expected behavior?
2. that didn't fail the step because error was in mid of pipeline. Consider using `shell: bash` to apply `-o pipefail` as well. See [SO answer](https://stackoverflow.com/a/75420766)